python/pyface3d/pyface3d.cxx

```diff
@@ -963,7 +963,7 @@ PYBIND11_MODULE(face3d, m)
 #endif
 
   py::class_<face3d::vertex_localizer>(m, "vertex_localizer")
-    .def(py::init<face3d::triangle_mesh const&>())
+    .def(py::init<face3d::triangle_mesh const&, int>())
```

Comment:
do you need to add a default argument here to avoid breaking existing python code?

Reply:
added